_usage/query_planning.md

```diff
@@ -0,0 +1,156 @@
+
+For example, the following command sequence turns on query planning, returns the query plan for the SELECT statement without executing the command, and then turns off query planning:
+
+```
+SET BABELFISH_SHOWPLAN_ALL ON
+GO
+SELECT b.bid, b.bbalance, t.tid, t.tbalance FROM pgbench_branches b, pgbench_tellers t WHERE b.bid=t.bid ORDER BY b.bid;
+GO
+                                     QUERY PLAN                                      
+-------------------------------------------------------------------------------------
+ Sort  (cost=37.96..39.21 rows=500 width=16)
+   Sort Key: b.bid
+   ->  Hash Join  (cost=3.12..15.55 rows=500 width=16)
+         Hash Cond: (t.bid = b.bid)
+         ->  Seq Scan on pgbench_tellers t  (cost=0.00..11.00 rows=500 width=12)
+         ->  Hash  (cost=2.50..2.50 rows=50 width=8)
+               ->  Seq Scan on pgbench_branches b  (cost=0.00..2.50 rows=50 width=8)
+(7 rows)
+
+SET BABELFISH_SHOWPLAN_ALL OFF
+GO
+```
+
+If you use the BABELFISH_STATISTICS PROFILE function, Babelfish will display the query plan that provides the best performance as it invokes the SELECT statement.  For example, the following command sequence returns an estimated cost to execute the SELECT statement and then turns off query planning:
+
+```
+SET BABELFISH_STATISTICS PROFILE ON
+GO
+SELECT b.bid, b.bbalance, t.tid, t.tbalance FROM pgbench_branches b, pgbench_tellers t WHERE b.bid=t.bid ORDER BY b.bid;
+GO
+
+                                                           QUERY PLAN                                                           
+--------------------------------------------------------------------------------------------------------------------------------
+ Sort  (cost=37.96..39.21 rows=500 width=16) (actual time=13.295..13.345 rows=500 loops=1)
+   Sort Key: b.bid
+   Sort Method: quicksort  Memory: 48kB
+   ->  Hash Join  (cost=3.12..15.55 rows=500 width=16) (actual time=4.035..13.111 rows=500 loops=1)
+         Hash Cond: (t.bid = b.bid)
+         ->  Seq Scan on pgbench_tellers t  (cost=0.00..11.00 rows=500 width=12) (actual time=2.263..11.175 rows=500 loops=1)
+         ->  Hash  (cost=2.50..2.50 rows=50 width=8) (actual time=1.733..1.734 rows=50 loops=1)
+               Buckets: 1024  Batches: 1  Memory Usage: 10kB
+               ->  Seq Scan on pgbench_branches b  (cost=0.00..2.50 rows=50 width=8) (actual time=0.012..1.712 rows=50 loops=1)
+ Planning Time: 0.218 ms
+ Execution Time: 13.452 ms
+(11 rows)
+
+SET BABELFISH_STATISTICS PROFILE OFF
+```
+
+
+**Query Plan display options**
+
+You can use parameter settings that are supported by the PostgreSQL [EXPLAIN and EXPLAIN ANALYZE statements](https://www.postgresql.org/docs/current/sql-explain.html) to control the type of information that is displayed with your query plan.  To query Babelfish for a list of parameters and their current values, use the following command:
```

Comment:
'that are supported by the PostgreSQL' : the settings shown are not identical to PG. Better to say they are 'similar' to PG

Reply:
Now reads:  You can use settings that are similar to the PostgreSQL...